feat: tutorial for btcd simnet (#51)
gonative-cc/relayer: contrib/README.md

Reviewer: Let's rename this file to `Bitcoin_mock.md`

# Run btcd simnet mode

## Install btcd and btcwallet

Follow the instruction in [btcd README.md](https://github.com/btcsuite/btcd?tab=readme-ov-file#installation)

> and [btcwallet README](https://github.com/btcsuite/btcwallet?tab=readme-ov-file#installation-and-updating) for install btcd and btcwallet

Reviewer: ```suggestion
and [btcwallet README](https://github.com/btcsuite/btcwallet?tab=readme-ov-file#installation-and-updating) to install btcd and btcwallet
```

## Run btcd with simnet mode

### Bootstrap the wallet and btc node

> First you need start the btcd in simnet mode

Reviewer: ```suggestion
First you need to start the btcd in simnet mode
```

```bash
btcd --simnet --rpcuser=user --rpcpass=password
```

> Create btc wallet. Please remember password we can use this somewhere.

Reviewer: ```suggestion
Create btc wallet. Please remember the password.
```

```bash
btcwallet --simnet --create
```

Connect you wallet to btcd simmode. The password and user here must be the btcd password/user pair.

> Open a new terminal and run command below:

Reviewer: ```suggestion
Open a new terminal and run:
```

```bash
btcwallet --simnet -u=user -P=password
```

> Create a new address we will use this for `--miningaddr`, the address receive btc when mining a new block.

Reviewer: ```suggestion
Create a new address. We will use it as the `--miningaddr` parameter - an address that receives btc when mining a new block.
```

> Open a new terminal and run command below:

Reviewer: ```suggestion
Open a new terminal and run:
```

```bash
btcctl --simnet --wallet --rpcuser=user --rpcpass=password getnewaddress
```

> Copy the address created by command above. Shutdown the btcd service and run with command below:

Reviewer: ```suggestion
Copy the address created by the command above. Shutdown the btcd service and run:
```
